entries/jQuery.parseHTML.xml

```diff
@@ -17,6 +17,7 @@
   <longdesc>
     <p><code>jQuery.parseHTML</code> uses a native DOM element creation function to convert the string to a set of DOM elements, which can then be inserted into the document.</p>
```

Comment:
@gibson042 I noticed it but didn't change it as it was unrelated to this PR. I updated it now.